assets/javascripts/discourse/templates/admin/plugins-explorer.hbs

```diff
@@ -147,9 +147,18 @@
       {{#if sortedQueries.length}}
         <table class="recent-queries">
           <thead class="heading-container">
-            <th class="col heading name">{{i18n 'explorer.query_name'}}</th>
-            <th class="col heading user">{{i18n 'explorer.query_user'}}</th>
-            <th class="col heading time">{{i18n 'explorer.query_time'}}</th>
+            <th {{action "sortByProperty" "name"}} class="col heading name">
+              <i class="fa fa-sort"></i>
```

Comment:
We should use `d-icon` here and below. For reference: https://github.com/discourse/discourse/blob/637850d8672855574aea8a43e5e1c1ad40ce0861/app/assets/javascripts/discourse/templates/user.hbs#L48

Reply:
Done, won't use `<i>` again, thanks! :grin: